File: static/code/sample/tethered/gdl-mcstick-a.cs
// In this sample:
// Toggle the statled every 500ms 

using GHIElectronics.DUELink;
using System;

var availablePort = DUELinkController.GetConnectionPort();
var duelink = new DUELinkController(availablePort);

void SetLed() {  
    duelink.Engine.ExecuteCommand("statled(500,500,0)");
}

SetLed();

File: static/code/sample/tethered/gdl-mcstick-a.py
# In this sample:
# Toggle the status LED every 500 ms
import time
from DUELink.DUELinkController import DUELinkController

availablePort = DUELinkController.GetConnectionPort()
duelink = DUELinkController(availablePort)

def SetLed():
    duelink.Engine.ExecuteCommand("statled(500,500,0)")

SetLed()